spec/lib/publishing_api_finder_publisher_spec.rb

```diff
@@ -2,6 +2,12 @@
 require "publishing_api_finder_publisher"
 
 describe PublishingApiFinderPublisher do
+  before do
+    @schema_type = GovukContentSchemaTestHelpers.configuration.schema_type
+    GovukContentSchemaTestHelpers.configuration.schema_type = "publisher"
```

Comment:
This means we won't be able to remove the old v1 schemas from content schemas. We could also remove this test or `skip: true` it?